ios/GatewayManager.m

```diff
@@ -64,6 +65,45 @@ - (void)configureStripeGateway:(NSDictionary *_Nonnull)gatewayParameters
 #endif
 }
 
+
+- (void)savePaymentMethod:(NSString *)secret cardParams:(NSDictionary *)cardParams completion:(void (^)(NSString * _Nullable, NSError * _Nullable))completion
```

Comment:
Let's call `secret` `clientSecret` so it's clearer what it's being used for

Reply:
👍